docker-compose.yaml

```diff
@@ -53,8 +53,12 @@ services:
             - SQL_PASSWORD=${SQL_PASSWORD}
             - SQL_HOST=${SQL_HOST}
         ports:
+            - 5678:5678
             - 8000:8000
-        command: bash -c "python manage.py migrate && python manage.py bootstrap && python manage.py runserver 0.0.0.0:8000"
+        command: >
+            bash -c "python manage.py migrate &&
+                    python manage.py bootstrap &&
+                    ${DEBUG_COMMAND:-python manage.py runserver 0.0.0.0:8000}"
```

Comment:
So this part runs the debug command as configured in your `.env` file if configured and runs the normal startup command if there is no debug command configured.